.github/workflows/ci.yml

```diff
@@ -76,17 +67,17 @@ jobs:
         os: [ubuntu-latest, macos-latest, windows-latest]
         rust: [1.60.0]
 
-    name: Check / Test MSRV on ${{ matrix.os }}
+    name: Check/Test MSRV (${{ matrix.rust }}) on ${{ matrix.os }}
```

Comment:
We removed the explicit version tagging, as its a pain in the rear because we would have to update branch protection rules in a "turn off, merge, then turn back on" fashion.

Reply:
Oh, gotcha